lib/utils/pyhelp.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * This file is part of the Micro Python project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Colin Hogben
```

Comment:
If you just move code around, please keep the original copyright. If you add some noticeable code and functionality, add your copyright (we need to remind that to people regularly too).